greenpeace/planet4-master-theme
# PLANET-7445: Migrate Split Two Columns blocks to Columns blocks

Ref: https://jira.greenpeace.org/browse/PLANET-7445

<hr>

**DESCRIPTION:**
This PR migrates the Split Two Columns blocks to Columns blocks.

<hr>

**RELATED PR:**
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1261

<hr>

**TESTING INSTRUCTIONS:**
1. Run this PR in your local environment.
2. Add a Split Two Columns block to a post or page. it won't work on the backend because of this PR's changes but it's good enough to test the migration script. You can also directly copy/paste this in the code editor:

```

<div class="wp-block-planet4-blocks-split-two-columns"><div data-hydrate="planet4-blocks/split-two-columns" data-attributes="{&quot;version&quot;:2,&quot;select_issue&quot;:0,&quot;title&quot;:&quot;This is issue name&quot;,&quot;issue_description&quot;:&quot;And this is the description&quot;,&quot;issue_link_text&quot;:&quot;A test link&quot;,&quot;issue_link_path&quot;:&quot;https://www.greenpeace.org/&quot;,&quot;issue_image_id&quot;:0,&quot;issue_image_src&quot;:&quot;&quot;,&quot;issue_image_srcset&quot;:&quot;&quot;,&quot;issue_image_title&quot;:&quot;&quot;,&quot;focus_issue_image&quot;:&quot;50% 50%&quot;,&quot;select_tag&quot;:6,&quot;tag_name&quot;:&quot;Climate&quot;,&quot;tag_description&quot;:&quot;Our climate is sick but we have the power to heal it and ourselves, and create a greener more equal world.\r\rExtreme weather events, sea level rise and droughts are all symptoms of a global climate on the brink of breakdown.\r\rWe are at a crossroads. If we act now, as a global community, to keep temperature rise below catastrophic levels, we have a chance to re-stabilise our delicate climate.\r\rWe ca&quot;,&quot;tag_link&quot;:&quot;http://www.planet4.test/tag/climate/&quot;,&quot;button_text&quot;:&quot;Get involved&quot;,&quot;button_link&quot;:&quot;http://www.planet4.test/tag/climate/&quot;,&quot;tag_image_id&quot;:344,&quot;tag_image_src&quot;:&quot;&quot;,&quot;tag_image_srcset&quot;:&quot;&quot;,&quot;tag_image_title&quot;:&quot;&quot;,&quot;focus_tag_image&quot;:&quot;50% 50%&quot;,&quot;edited&quot;:{&quot;title&quot;:true,&quot;issue_description&quot;:true,&quot;issue_link_text&quot;:true,&quot;tag_description&quot;:false,&quot;button_text&quot;:false,&quot;issue_image_id&quot;:false,&quot;tag_image_id&quot;:false}}"><section class="alignfull split-two-column "><div class="split-two-column-item item--left"><div class="split-two-column-item-content"><h2 class="split-two-column-item-title"><a href="https://www.greenpeace.org/" data-ga-category="Split Two Columns" data-ga-action="Category Title" data-ga-label="n/a">This is issue name</a></h2><p class="split-two-column-item-subtitle">And this is the description</p><a class="split-two-column-item-link" href="https://www.greenpeace.org/" data-ga-category="Split Two Columns" data-ga-action="Text Link" data-ga-label="n/a">A test link</a></div></div><div class="split-two-column-item item--right"><div class="split-two-column-item-content"><a class="split-two-column-item-tag" href="http://www.planet4.test/tag/climate/" data-ga-category="Split Two Columns" data-ga-action="Tag Title" data-ga-label="n/a"><span aria-label="hashtag">#</span>Climate</a><p class="split-two-column-item-subtitle">Our climate is sick but we have the power to heal it and ourselves, and create a greener more equal world.

Extreme weather events, sea level rise and droughts are all symptoms of a global climate on the brink of breakdown.

We are at a crossroads. If we act now, as a global community, to keep temperature rise below catastrophic levels, we have a chance to re-stabilise our delicate climate.

We ca</p><a class="btn btn-primary split-two-column-item-button" href="http://www.planet4.test/tag/climate/" data-ga-category="Split Two Columns" data-ga-action="Call to Action" data-ga-label="n/a">Get involved</a></div></div></section></div></div>



<div class="wp-block-planet4-blocks-split-two-columns"><div data-hydrate="planet4-blocks/split-two-columns" data-attributes="{&quot;version&quot;:2,&quot;select_issue&quot;:0,&quot;title&quot;:&quot;&quot;,&quot;issue_description&quot;:&quot;&quot;,&quot;issue_link_text&quot;:&quot;&quot;,&quot;issue_link_path&quot;:&quot;&quot;,&quot;issue_image_id&quot;:0,&quot;issue_image_src&quot;:&quot;&quot;,&quot;issue_image_srcset&quot;:&quot;&quot;,&quot;issue_image_title&quot;:&quot;&quot;,&quot;focus_issue_image&quot;:&quot;50% 50%&quot;,&quot;select_tag&quot;:0,&quot;tag_name&quot;:&quot;&quot;,&quot;tag_description&quot;:&quot;&quot;,&quot;tag_link&quot;:&quot;&quot;,&quot;button_text&quot;:&quot;Get involved&quot;,&quot;button_link&quot;:&quot;&quot;,&quot;tag_image_id&quot;:0,&quot;tag_image_src&quot;:&quot;&quot;,&quot;tag_image_srcset&quot;:&quot;&quot;,&quot;tag_image_title&quot;:&quot;&quot;,&quot;focus_tag_image&quot;:&quot;50% 50%&quot;,&quot;edited&quot;:{&quot;title&quot;:false,&quot;issue_description&quot;:false,&quot;issue_link_text&quot;:false,&quot;tag_description&quot;:false,&quot;button_text&quot;:false,&quot;issue_image_id&quot;:false,&quot;tag_image_id&quot;:false}}"><section class="alignfull split-two-column "><div class="split-two-column-item item--left"><div class="split-two-column-item-content"></div></div><div class="split-two-column-item item--right"><div class="split-two-column-item-content"></div></div></section></div></div>

```

4. Run the migrator command: `npx wp-env run cli wp p4-run-activator`
5. Check whether the migration worked as expected.
6. To reset the environment database back to the latest default content and repeat the tests, run: `npm run db:reset`.
